comms/internal/rpcz/rate_limit.go

```diff
@@ -0,0 +1,30 @@
+package rpcz
+
+import (
+	"log"
+
+	"comms.audius.co/config"
+	"comms.audius.co/jetstream"
+	"github.com/nats-io/nats.go"
+)
+
+func SetupRateLimitRules() {
+	jsc := jetstream.GetJetstreamContext()
+	if jsc == nil {
+		log.Fatal("jetstream not ready")
+	}
+	kv, err := jsc.CreateKeyValue(&nats.KeyValueConfig{
+		Bucket:   config.RateLimitRulesBucketName,
+		Replicas: config.NatsReplicaCount,
+	})
+	if err != nil {
+		log.Fatal("CreateKeyValue failed ", err, " bucket: ", config.RateLimitRulesBucketName)
+	}
+
+	for rule, limit := range config.DefaultRateLimitRules {
```

Comment:
further down there's a note about adding a helper function for `getRateLimitWithFallback` or similar... if that function exists there's probably no need to populate the KV with the defaults in the first place, and we can just do a KV put if we need to modify the default.

If this code runs at boot it will overwrite any custom values with defaults on server boot... I think we write helper function and delete this.